WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderCommentListActivity.java

```diff
@@ -380,9 +397,17 @@ private void refreshComments() {
      * scrolls the passed comment to the top of the listView
      */
     private void scrollToCommentId(long commentId) {
+        scrollToCommentIdWithSmooth(commentId, false);
+    }
+
+    private void scrollToCommentIdWithSmooth(long commentId, boolean shouldSmooth) {
```

Comment:
Any chance we can rename this to `smoothScrollToCommentId()` without the `shouldSmooth` param?